include/revng/Model/FunctionEdge.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "revng/Model/FunctionEdgeType.h"
+#include "revng/Model/GeneratedFunctionEdge.h"
+
+/// An edge on the CFG
+class model::FunctionEdge : public model::generated::FunctionEdge {
+public:
+  using generated::FunctionEdge::FunctionEdge;
+
+public:
+  bool operator<(const FunctionEdge &Other) const {
+    auto ThisTie = std::tie(Destination, Type);
+    auto OtherTie = std::tie(Other.Destination, Other.Type);
+    return ThisTie < OtherTie;
+  }
```

Comment:
Can you try to generate the `<=>` operator?

Reply:
I tried (with Pietro help) but failed because (likely) our libcpp does not implement `operator<=>` for `std::tuple`